src/app/components/hmcts-global-header/hmcts-global-header.component.html

```diff
@@ -69,11 +69,11 @@
         <ul class="hmcts-primary-navigation__list">
           <li class="hmcts-primary-navigation__item" *ngFor="let item of (rightItems | async)">
             <a class="hmcts-primary-navigation__link transparent-background" [ngClass]="item.ngClass" (click)="onEmitSubMenu(item)"
-            [attr.aria-current]="item.active ? true : false" [routerLink]="item.href" *ngIf="showFindCase && item.ngClass === 'hmcts-search-toggle__button'">{{item.text}}</a>
+            [attr.aria-current]="item.active ? true : null" [routerLink]="item.href" *ngIf="item.ngClass === 'hmcts-search-toggle__button'">{{item.text}}</a>
             <exui-case-reference-search-box
+              *ngIf="item.ngClass !== 'hmcts-search-toggle__button'"
```

Comment:
As above. Wouldn't it be better to check the state of the feature toggle? (A directive for this is available in the common lib)

Reply:
Thank you @ronaldmansveld - I have now implemented the changes.